tap_exchangeratesapi/__init__.py

```diff
@@ -90,13 +126,29 @@ def do_sync(base, start_date):
 def main():
     parser = argparse.ArgumentParser()
 
+    parser.add_argument(
+        '-d', '--discover', help='Do schema discovery', required=False, action='store_true')
```

Comment:
I guess it never said config and state are also optional, so this might be moot.

Reply:
@dmosorast - I see your point and I agree - specifically because we're bending the rules a little bit about what is/isn't required.

I'll update the help text to explicitly mention that these optional.

Reply:
Done! ✅